metadata.rb

```diff
@@ -24,3 +24,5 @@
 depends 'selinux', '~> 3.1.1'
 depends 'yum', '~> 6.1.1'
 depends 'yum-epel', '~> 4.1.2'
+depends 'aws-parallelcluster-install', '~> 3.0.0'
```

Comment:
remember to change the util to bump the cookbook version, to bump aws-parallelcluster-install and aws-parallelcluster-config as well

Reply:
I think I got this -- might help to have a spot-check on it though.